package.json

```diff
@@ -81,6 +83,7 @@
     "karma-webpack": "=2.0.3",
     "mocha": "=3.2.0",
     "postcss-loader": "=2.0.6",
+    "postcss-nested": "=2.0.2",
```

Comment:
we have already used cssnext plugin in our postcss config and it has nesting feature.So,there is no reason to use another plugin to add nesting feature to project.Please remove 'postcss-nested' and use cssnext nesting feature instead of that.I suggest to read cssnext features again.You can find more information here about nesting in cssnext in this link: http://cssnext.io/features/#nesting